site/resources.md

```diff
@@ -33,19 +33,19 @@ Resources
   _add description_
 
 ### Data Analysis
-* [Numpy (basics)] (https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/numpy.ipynb)
+* [Numpy (basics)](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/numpy.ipynb)
```

Comment:
For these would it be possible to use bold for the titles and use slightly different spacing for the description:

`* [**Numpy (basics)**](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/numpy.ipynb)  `

with 2 spaces at the end of the link, and then the description starting on the immediate next line (see python-awips examples further down in this document)

Reply:
Got it - thanks!

Reply:
See the newest review of the latest commit below.